pkg/utils/utils.go

```diff
@@ -460,10 +474,15 @@ func setVfsAdminMac(iface *sriovnetworkv1.InterfaceExt) error {
 			glog.Errorf("setVfsAdminMac(): unable to get VF id %+v %q", iface.PciAddress, err)
 			return err
 		}
+		_, err = vfIsReady(addr)
+		if err != nil {
+			glog.Errorf("setVfsAdminMac(): VF link is not ready for device %+v %q", addr, err)
+			return err
+		}
 		vfName := tryGetInterfaceName(addr)
```

Comment:
As we've got the vf name in `vfIsReady`, can we remove this line?  And also for the `vfLink` line bellow.

Reply:
fixed